### ISSUE TITLE
`start <ID found by completion>` don't find the ID

### ISSUE THREAD

**Issue by luffah** [soywod/unfog#43]
Hello, i got a small issue with the completion : the id returned is not recognized by `unfog start` command.
```sh
# system : ubuntu 18.04
# installation method : install.sh
# version: 1.0.1

·▓█  ~/apps █ unfog list

ID   |DESC           |PROJECT     |ACTIVE |DUE |WORKTIME 
a4fb |Test module PR |helpdeskv14 |       |    |3 mins   

# typing `unfog start a<Tab>`, it is completed by bash with an ID
·▓█ ?=130 ~/apps █ unfog start a4fbcc5e-b391-4f60-9e0a-46375a37d73f 
Error: task not found
```

**Comment by soywod** [comment]
This is already fixed on master, I was waiting for #40 to release the next version. I let you know !

**Comment by soywod** [comment]
Fixed in https://github.com/soywod/unfog/releases/tag/v1.0.2.
